segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StorageEventProcessor.java

```diff
@@ -43,6 +43,9 @@
 public class StorageEventProcessor implements AbstractTaskQueueManager<GarbageCollector.TaskInfo> {
 
     private final static GarbageCollector.TaskInfo.Serializer SERIALIZER = new GarbageCollector.TaskInfo.Serializer();
+    // For SLTS GC, truncating the ContainerEventProcessor internal Segment every 1MB is enough to prevent unnecessary
+    // storage metadata activity that may occur if we truncate it too frequently.
+    private final static long CONTAINER_EVENT_PROCESSOR_TRUNCATE_DATA_SIZE = 1024 * 1024;
```

Comment:
I wonder if this could be config value.

Reply:
I mentioned this in the PR: It is configurable at the `ContainerEventProcessor` level. But, really, this is going to be hardly used by anyone, so I decided to make it a constant from the "client", which is the `StorageEventProcessor`. I could expose this in the same way as `maxItemsAtOnce`, but I just don't see the value of it. I can do it anyway if you feel that it is better that way.

Reply:
I guess that's fine for now.